### PR TITLE
Assignment Done

### PR DESCRIPTION
**Pull request by PiyushPJun20**
Assignment_Done_1Hour

### CODE REVIEW

File: src/MaxMin.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+public class MaxMin {
+	
+	void maximum(int[] input) {
+		int max = input[0];
+		for(int index=0; index<input.length; index++) {
```
Comment (KrishnaKTechnocredits) [inline src/MaxMin.java]: input[0] is already in max. you should start loop from index 1

File: src/MaxMin.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			if(input[index]>max)
+				max = input[index];
+		}
+		System.out.println(" Maximum Number is: " + max);
+	}
+	
+	void minimum(int[] input) {
+		int min = input[0];
+		for(int index=0; index<input.length; index++) {
```
Comment (KrishnaKTechnocredits) [inline src/MaxMin.java]: same here.

File: src/NameLength.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class NameLength {
+	
+	void printLength(String[] input) {
+		String max = input[0];
+		for(int index=0; index<input.length;index++) {
```
Comment (KrishnaKTechnocredits) [inline src/NameLength.java]: same here.

File: src/MaxMin.java (hunk 1 above)
Comment (PiyushPJun20) [reply src/MaxMin.java]: Changes Incorporated.

File: src/MaxMin.java (hunk 2 above)
Comment (PiyushPJun20) [reply src/MaxMin.java]: Changes Incorporated.

File: src/NameLength.java (hunk 3 above)
Comment (PiyushPJun20) [reply src/NameLength.java]: Changes Incorporated. Thanks.

**Pull request merged**